modules/ossm-enabling-jaeger.adoc

```diff
@@ -22,6 +22,13 @@ spec:
     type: Jaeger
 ----
 
-Currently the only tracing type that is supported is `Jaeger`.   Jaeger is enabled by default.  To disable tracing, set `type` to `None`.
+Currently the only tracing type that is supported is `Jaeger`.
 
-The sampling rate determines how often a trace is generated. You configure `sampling` as a scaled integer representing 0.01% increments.  For example, setting the value to `10` samples 0.1% of traces, setting the value to `500` samples 5% of traces, and a setting of `10000` samples 100% of traces.
+Jaeger is enabled by default.  To disable tracing, set `type` to `None`.
+
+The sampling rate determines how often the Envoy proxy generates a trace. You can use the sampling rate option to control what percentage of requests get reported to your tracing system. You should configure this setting based upon your traffic in the mesh and the amount of tracing data you want to collect. You configure `sampling` as a scaled integer representing 0.01% increments.  For example, setting the value to `10` samples 0.1% of traces, setting the value to `500` samples 5% of traces, and a setting of `10000` samples 100% of traces.
```

Comment:
s/should/must (or "can" if that's more appropriate but point being, "should" should be avoided if possible 😀 )